CactbotOverlay/JSEvents.cs

```diff
@@ -72,6 +72,18 @@ public class PartyWipeEvent : JSEvent {
       public string EventName() { return "onPartyWipe"; }
     }
 
+    public class StatusEffectsUpdateEvent : JSEvent {
+      public StatusEffectsUpdateEvent(FFXIVProcess.EntityData player, FFXIVProcess.EntityData target, FFXIVProcess.EntityData focus) {
+        playerStatusEffects = player?.statusEffects;
+        targetStatusEffects = target?.statusEffects;
+        focusStatusEffects = focus?.statusEffects;
```

Comment:
Do you think it'd be worth optimizing the case where target and focus are the same?